File: simulation_prototype/Drink.py
from Product import Product


class Drink(Product):
    def __init__(self, new_id, new_name, new_price, new_drinkingTime):
        super().__init__(new_id, new_name, new_price, 0)

        self._drinkingTime = new_drinkingTime


    def __str__(self):
        return "NAME:" + self._name


    def getEatingTime(self):
        return self._drinkingTime

File: Drink.cpp
#include "Drink.h"

#include <iostream>

Drink::Drink(int new_id, std::string new_name, double new_price, 
            int new_drinkingTime): Product(new_id, new_name, new_price, 0)
{
    drinkingTime = new_drinkingTime;
}

int Drink::getEatingTime()
{
    return drinkingTime;
}

std::ostream& operator<<(std::ostream& os, const Drink& drink)
{
    os << drink.ID << " | " << drink.name << " | " << drink.price << std::endl;
    return os;
}

Drink::~Drink()
{
    
}